### DISCUSSION TITLE
ART 1.14.1

### DISCUSSION THREAD

**Discussion by beat-buesser** [Trusted-AI/adversarial-robustness-toolbox#2122] [Announcements]
This release of ART 1.14.1 provides updates to ART 1.14

# Added

[None]

# Changed

[None]

# Removed

[None]

# Fixed

- Fixed bug in `PytorchYolo` object detection estimator to correctly normalize the bounding boxes (#2091)
- Fixed missing `adversarial_accuracy` metric in `__init__.py` (#2093 )
- Fixed bug of default value for a loss weighting parameter being used rather than user supplied inputs in `AdversarialTrainerCertifiedIBPPyTorch` (#2112)
- Fixed Regional Misclassification Attack (RMA) to be able to poison all bounding boxes regardless of the class type (#2110 )
- Fixed wrong order of predictions and targets arguments in `AutoProjectedGradientDescent`'s new cross entropy loss class introduced in ART 1.14.0 and ensured correct attributes in `PyTorchClassifier` (#2117)

<hr /><em>This discussion was created from the release <a href='https://github.com/Trusted-AI/adversarial-robustness-toolbox/releases/tag/1.14.1'>ART 1.14.1</a>.</em>
